modules/navier_stokes/doc/content/source/auxkernels/INSQCriterionAux.md

```diff
@@ -0,0 +1,24 @@
+# INSQCriterionAux
+
+This calculates a scalar variable calculated as a function of the shear rate
+properties of a fluid flow called $Q$ which is useful for visualizing complicated
+flow patterns which tend to occur when running turbulent fluid flow simulations.
+The quantity is calculated as:
+
+\begin{equation}
+Q = \frac{1}{2} (|| \bar\bar \Omega || - ||\bar\bar S ||)
```

Comment:
@aeslaughter do you know why this isn't rendering correctly [here](https://mooseframework.inl.gov/docs/PRs/15758/site/source/auxkernels/INSQCriterionAux.html)? Just googling LaTeX, I would be inclined to think we need to be doing `\bar{}` instead of just `\bar`

Reply:
Ah, yeah, it definitely should be wrapped in curlies.